test/running_modules/run_tests

```diff
@@ -0,0 +1,48 @@
+#/usr/bin/env sh
+
+should_succeed() {
+    echo Running: "$@"
+    eval "$@"
+    if [ $? -ne 0 ]
+    then
+        echo ERROR: Command should have succeeded
+        exit 1
+    else
+        echo Test Passed
+    fi
+}
+should_fail() {
+    echo Running: "$@"
+    eval "$@"
+    if [ $? -eq 0 ]
+    then
+        echo ERROR: Command should have failed
+        exit 1
+    else
+        echo Test Passed
+    fi
+}
+
+# No module
+should_succeed cargo run -- run --target erlang
+should_succeed cargo run -- run --target javascript --runtime nodejs
+should_succeed cargo run -- run --target javascript --runtime deno
+
+# Module from same package
+should_succeed cargo run -- run --module module --target erlang
+should_succeed cargo run -- run --module module --target javascript --runtime nodejs
+should_succeed cargo run -- run --module module --target javascript --runtime deno
+
+# Nested module from same package
+should_succeed cargo run -- run --module module/sub_module --target erlang
+should_succeed cargo run -- run --module module/sub_module --target javascript --runtime nodejs
+should_succeed cargo run -- run --module module/sub_module --target javascript --runtime deno
```

Comment:
Could we get a test for a module from a different package? To ensure the JavaScript import logic is working correctly. Could add `gleeunit` as a dep, that has a main function.

Reply:
done